### PR TITLE
Export new concentration field

### PR DESCRIPTION
**Pull request by islean**
## Description

As part of the bypass qc flow https://github.com/Clinical-Genomics/project-planning/issues/485, a new concentration field for samples have been added. So far this concentration is parsed from the order form in the new order portal via #2740 and https://github.com/Clinical-Genomics/clinical-genomics-ui/pull/375. This PR also makes sure that the new field is exported to LIMS when submitting the order.

### Fixed

- New concentration is uploaded to LIMS

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by islean** [comment]
New field (UDF/ Concentration (ng/ul)) parsed correctly: https://clinical-lims-stage.scilifelab.se/api/v2/samples/ACC13411A1

**Comment by islean** [comment]
New field (UDF/ Concentration (ng/ul)) parsed correctly: https://clinical-lims-stage.scilifelab.se/api/v2/samples/ACC13411A1

File: cg/constants/lims.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "comment": "Comment",
     "control": "Control",
     "concentration": "Concentration (nM)",
+    "concentration_ng_ul": "Concentration (ng/ul)",
```
Comment (henrikstranneheim) [inline cg/constants/lims.py:15]: We should refactor this to a pydantic model in another PR

File: cg/meta/orders/case_submitter.py [hunk 2]
```diff
@@ -195,6 +195,7 @@ def order_to_status(order: OrderIn) -> dict:
                         "application": sample.application,
                         "capture_kit": sample.capture_kit,
                         "comment": sample.comment,
+                        "concentration_ng_ul": sample.concentration_ng_ul,
```
Comment (henrikstranneheim) [inline cg/meta/orders/case_submitter.py:198]: And here as well

**Review by henrikstranneheim — APPROVED** [review]
👍

**Review by diitaz93 — APPROVED** [review]
LGTM 👍

File: cg/constants/lims.py (hunk 1 above)
Comment (islean) [reply cg/constants/lims.py:15]: I would like to refactor quite a bit in the LIMS module...

**Comment by islean** [comment]
Deployed to Hasta and cg-prod-services:

```
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/isak.ohlsson
done.
Log deploy... done.
cg, version 54.10.6
θ60° 1d [isak.ohlsson@hasta:~] [P_base] 3m10s $ 
```

**Pull request merged**